Component/Console/Style/ImagineStyleInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Component\Console\Style;
+
+/**
+ * @internal
+ */
+interface ImagineStyleInterface
```

Comment:
can we re-validate which of these are actually used (outside of the test)? i have the impression only half of these are actually used. we should not define more than we need at the moment, and add them as we go. the interface is marked internal, so adding method seems possible. (not sure if we even need an interface or should rather just have the concrete class)

Reply:
I felt the same about the interface, I'll take a look an see what can I do